### PR TITLE
Fix/ve 610 merge save return

### PR DESCRIPTION
**Pull request by stratease**
### 🎫 Ticket

[BTRIA-2310] and [VE-610]

### 🗒️ Description

The ORM Decorator is not returning results from all of it's containerized methods. This adds return results, namely for the `save()` method. Solves a bug found in https://github.com/the-events-calendar/events-pro/actions/runs/9085591273/job/24969287543?pr=2444.

### 🎥 Artifacts 

image

### ✔️ Checklist
- [x] Changelog entry in the `readme.txt` file.
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [x] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [x] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [x] Have you added Artifacts?
- [x] Check the base branch for your PR.
- [x] Add your PR to the project board for the release.

[BTRIA-2310]: https://stellarwp.atlassian.net/browse/BTRIA-2310?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[VE-610]: https://stellarwp.atlassian.net/browse/VE-610?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by stratease** [comment]
This PR is big because the bucket is outdated from master.

**Comment by Camwyn** [comment]
> This PR is big because the bucket is outdated from master.

@stratease Fixed that for you!

File: src/Tribe/Repository/Decorator.php [hunk 1]
```diff
@@ -486,7 +486,7 @@ public function filter_postarr_for_create( array $postarr ) {
 	 * {@inheritdoc}
 	 */
 	public function set_create_args( array $create_args ) {
-		$this->decorated->set_create_args( $create_args );
+		return $this->decorated->set_create_args( $create_args );
 	}
```
Comment (rafsuntaskin) [inline src/Tribe/Repository/Decorator.php:490]: According to the inherited doc there are no returns, should we update the Docblock in the parent? Or just update within decorator?

Comment (stratease) [reply src/Tribe/Repository/Decorator.php:490]: I'll update the interface docs, good eye thanks!

**Review by rafsuntaskin — APPROVED** [review]

**Review by bordoni — APPROVED** [review]
@lucatume can we get some feedback on this change too?

**Pull request merged**